docs/vectorizer.md

```diff
@@ -270,698 +296,700 @@ SELECT ai.create_vectorizer(
+|vectorizer_id| int  | -       |✔| The identifier of the vectorizer whose schedule you want to enable. |
 
-3. These functions are particularly useful in scenarios such as:
-    - System maintenance windows where you want to reduce database load
-    - Temporarily pausing processing during data migrations or large bulk updates
-    - Debugging or troubleshooting issues related to the vectorizer
-    - Implementing manual control over when embeddings are updated
+#### Returns
 
-4. When a schedule is disabled, new or updated data will not be automatically processed. However, the data will still be queued, and will be processed when the schedule is re-enabled or when the vectorizer is manually run.
+IAIN: I don't know what this function returns.
 
-5. These functions only affect vectorizers that use scheduled processing (i.e., those configured with ai.scheduling_timescaledb). Vectorizers configured with ai.scheduling_none are not affected.
+### ai.disable_vectorizer_schedule
 
-6. After re-enabling a schedule, the next run will occur based on the original scheduling configuration (e.g., if it was set to run every hour, it will run at the next hour mark after being enabled).
+You use `ai.disable_vectorizer_schedule` to:
+- Deactivate the scheduled job for a specific vectorizer.
+- Temporarily stop the automatic processing of new or updated data.
 
-Usage example in a maintenance scenario:
 
-```sql
--- Before starting system maintenance
-SELECT ai.disable_vectorizer_schedule(1);
-SELECT ai.disable_vectorizer_schedule(2);
+#### Example usage
 
--- Perform maintenance tasks...
+To stop the automatic scheduling for the vectorizer with ID 1.
 
--- After maintenance is complete
+```sql
 SELECT ai.enable_vectorizer_schedule(1);
-SELECT ai.enable_vectorizer_schedule(2);
 ```
 
-These functions provide an important layer of operational control for managing 
-pgai vectorizers in production environments. They allow database administrators 
-and application developers to balance the need for up-to-date embeddings with 
-other system priorities and constraints, enhancing the overall flexibility and 
-manageability of the pgai extension.
+#### Parameters
+
+`ai.enable_vectorizer_schedule` takes the following parameters:
+
+|Name| Type | Default | Required | Description                                                          |
+|-|------|---------|-|----------------------------------------------------------------------|
+|vectorizer_id| int  | -       |✔| The identifier of the vectorizer whose schedule you want to disable. |
+
+#### Returns
+
+IAIN: I don't know what this function returns.
 
 ## Processing configuration
 
-The processing configuration functions in the pgai extension are used to specify 
-how the vectorizer should process data when generating embeddings. These 
-functions allow users to choose between different processing strategies, 
+You use the processing configuration functions in pgai to specify 
```

Comment:
This whole section is out of date, but I think it may be easier to merge as-is and then submit fixes.

Reply:
Updated, thank you